PowerNSX.psm1

```diff
@@ -15691,6 +15694,24 @@ function Set-NsxEdgeOspf {
     The Set-NsxEdgeOspf cmdlet allows manipulation of the OSPF specific
     configuration of a given ESG.
 
+    .EXAMPLE
```

Comment:
Issue doesnt relate to GR and Default Originate.  Can you remove from this PR?  They need to be updated with the changes that I still hvae half done for fixing set-nesxedge to allow toggle of both on an already configured edge....